modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/home/components/ProjectsList/ProjectList.test.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {faker} from '@faker-js/faker';
```

Comment:
No need for faker, inside a unit test

When we develop a unit test, it should be simply as possible, and easily readable to anyone. Also using a library increases the node_modules size, and worse, can create a dependency of his use.

I know, probably there's no relationship with this library in particular, but the faker I used to use, caused many problems to many users, "encouraged as a protest" to Ukraine War

I'm not saying it will happen again, but as much we can avoid adding external libraries, better 😄 

References:

https://stackoverflow.com/questions/70597019/what-happened-with-faker-js
https://www.infoworld.com/article/3647750/faker-npm-package-back-on-track-after-malicious-coding-incident.html